requirements-dev.txt

```diff
@@ -10,3 +10,4 @@ semver
 pyee
 requests
 qrcode
+apscheduler
```

Comment:
Because I have created `DailyPlugin` and `DingDongRankPlugin` which use apscheduler, so, this package dependency item is required. After We move it to `python-wechaty-plugin-contrib` package, I will remove it `requirement-dev.txt` file.